_drafts/oryx/setup.md

```diff
@@ -0,0 +1,37 @@
+# Setup
+
+This guide explains how to set up your environment for Oryx framework development.
+
+## Prerequisites
+
+To install Oryx on your local system, you need the following:
+
+- [Node.js](https://nodejs.org/) or compatible Javascript runtime
+- npm package manager
+- A [Spryker storefront API](https://docs.spryker.com/docs/scos/dev/glue-api-guides/202204.0/glue-rest-api.html) (a public API is provided by default to get you up and running quick)
```

Comment:
How do I fulfill this requirement?

Reply:
There are several possible ways: use publicly available API or install Spryker instance locally. We want to provide our boilerplate with default public API so users can skip this step at the beginning. We can basically add a link to the Spryker Installation guide :)

Reply:
Since the only way we describe the installation is using the boilerplate, does it make sense to put it as a requirement?

Reply:
You mean to add a boilerplate as a requirement?

Reply:
I mean to remove this requirement because the API is included with the boilerplate

Reply:
Not really sure. The boilerplate is just a recommended and easiest way to start work with Oryx, but not the only one.

Reply:
No, let's keep it with the installation guide. If there's is a Spryker documentation on the GLUE API we could link to it would be great.